src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/MetaDescription.js

```diff
@@ -40,6 +41,10 @@ export default connect()(function MetaDescription({
       }
 
       setError(message);
+
+      if (isSaving) {
```

Comment:
yes for now it's relying on the useEffect since everytime the user is saving that's the only time I can rerun the onChange function to check the characters condition on excess.

Reply:
@agalin920 I removed now the useEffect on every component and let the useEffect inside of ItemSettings do the work of removing the errors while on there are on saving... Then i slice all the props base on the number of allowed character on the properties in the /shell/store/content.js